samples/matter/light_bulb/sample.yaml

```
@@ -2,24 +2,20 @@ sample:
   description: Matter Light Bulb example
   name: Matter Light Bulb
 tests:
+  # Excluded in quarantine.yaml to limit resources usage in integration builds
   sample.matter.light_bulb.debug:
     build_only: true
+    extra_args: CONFIG_CHIP_LIB_SHELL=y
     integration_platforms:
       - nrf52840dk_nrf52840
       - nrf5340dk_nrf5340_cpuapp
-      - nrf7002dk_nrf5340_cpuapp
     platform_allow: nrf52840dk_nrf52840 nrf5340dk_nrf5340_cpuapp nrf7002dk_nrf5340_cpuapp
-    tags: ci_build
-  sample.matter.light_bulb.no_dfu:
+  sample.matter.light_bulb.wifi:
```

```suggestion
  sample.matter.light_bulb.debug_wifi:
```
or `debug.wifi` similarly to `debug.nrf5340dk` you have below? Just in case we need other wifi tests

---

OK, but maybe I should also change `wifi` to 'nrf7002dk'?

---

Fine with me.